Upstream-Status: Pending

Nitin A Kamble <nitin.a.kamble@intel.com> 2011/05/10
Fix following build error:

| NOTE: make -j 16
| Makefile:719: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.
| ERROR: oe_runmake failed

Index: git/Makefile.am
===================================================================
--- git.orig/Makefile.am
+++ git/Makefile.am
@@ -5,5 +5,5 @@ SUBDIRS = src data
 MAINTAINERCLEANFILES = aclocal.m4 compile config.guess config.sub configure depcomp install-sh ltmain.sh Makefile.in missing
 
 snapshot:
-        $(MAKE) dist distdir=$(PACKAGE)-snap`date +"%Y%m%d"`
+	$(MAKE) dist distdir=$(PACKAGE)-snap`date +"%Y%m%d"`
 
